nbsite/pyodide/__init__.py

```diff
@@ -177,6 +182,50 @@ def _model_json(model: Model, target: str) -> Tuple[Document, str]:
         version   = BOKEH_VERSION,
     ))
 
+def write_resources(out_dir, source, resources):
+    """
+    Writes resources extracted from process to a shared JSON file to
+    which will be read on document write. On windows uses in-memory
+    variable since parallelization is not supported.
+
+    Arguments
+    ---------
+    out_dir: str
+        The build directory
+    source: str
+        The source file the resources are assosicated with.
+    resources: dict[str, any]
+        The resources to add to the resource dictionary.
+    """
+    out_path = pathlib.Path(str(out_dir))
```

Comment:
```suggestion
    out_path = pathlib.Path(os.fspath(out_dir))
```

However, I don't think the conversion is needed. If it is the docstring should be updated.